src/practices/TypeScript/TsGitignoreCorrectlySetPractice.ts

```diff
@@ -12,7 +13,7 @@ import { PracticeContext } from '../../contexts/practice/PracticeContext';
   url: 'https://github.com/github/gitignore/blob/master/Node.gitignore',
```

Comment:
We should use another suggested .gitignore as they don't ignore lock files and we require it.

Reply:
There will be an article in a Knowledge Base.

Reply:
Will it be in days? Or?